docs/configuration-ini.md

```diff
@@ -281,7 +281,27 @@ Start with a "[ ${layer name} ]" which must be unique throughtout the model. In
 
 2. ```trainable = <bool>```
 
-   If this backbone must be trained. Only supported for ini backbones (nntrainer models).
+   If this backbone must be trained (defaults to false). Only supported for ini backbones (nntrainer models).
+
+3. ```Preload = <bool>```
+
+   Load pretrained weights from the saved modelfile of backbonE (defaults to false). Only supported for ini backbone (nntrainer models).
+
+4. ```ScaleSize = <float>```
+
+   Scale the size of the layers from backbone (defaults to 1.0). This applies for fully connected and convolution layer for now, where the units and the output channels are scaled respectively. Only supported for ini backbone (nntrainer models).
```

Comment:
I wonder that if developer want to reduce the weight which is already trained, then how it works? I mean, let's say the filter size of conv layer is 10 and pre-loaded. If he set the scale 0.5, then how it works?

Reply:
Yes, I think this is a very important issue which should have been made clear in the documentation. 
Only one of preload or scaling can be done. Both the options activated together cannot be done for now.
Updated this to documentation.

Reply:
> Yes, I think this is a very important issue which should have been made clear in the documentation.
> Only one of preload or scaling can be done. Both the options activated together cannot be done for now.
> Updated this to documentation.

Then it means scaling is only validate for the backbone network which is ready for training, then. Cause even if it is trained already, we can use scale, if we are going to train the backbone. Or If we are not going to train, then it is not allowed.

Reply:
Exactly.